packaging/helm/trento-server/charts/trento-web/values.yaml

```diff
@@ -3,12 +3,11 @@
 # Declare variables to be passed into your templates.
 
 replicaCount: 1
-  
+
 image:
-  repository: ghcr.io/trento-project/trento
-  pullPolicy: IfNotPresent
-  # Overrides the image tag whose default is the chart appVersion.
-  tag: ""
+  repository: ghcr.io/trento-project/trento-web
+  pullPolicy: Always
+  tag: "rolling"
```

Comment:
Could we avoid this? ._.

Reply:
hmm, what do you mean?

Reply:
I guess @dottorblaster means to ask if we could avoid the `pullPolicy: Always`? If so, I understood this is a side effect of using `tag: rolling` (to ensure the image gets pulled even when the tag is the same).

Reply:
I mean, for now I think it's useful to have an `Always` pullPolicy and point to the rolling tag but keep in mind that this will end up in the hands of customers, and I don't know if this is the best strategy to handle the distribution of the containers :slightly_smiling_face:

@rtorrero anticipated my answer, Github was down while I was posting this comment :D

Reply:
Maybe I'm missing something, but I don't see anything particularly problematic about `Always`: the container engine will not waste bandwidth if the image hash is identical, but by using `Always` we at least enforce to try to see if there is a new image available, otherwise it will keep using whatever it has already in cache under that repo:tag.
TL;DR: I'm not aware of good reasons not to use it, even for non-rolling tags, and it will surely avoid upgrading headaches.